tests/e2e/core-tests/group_02_product_func_test.js

```diff
@@ -2,10 +2,11 @@ var faker = require('faker');
 const { fake } = require('faker');
 const helpers = require('../pages/helpers');
 Feature('Group Product Functionality Test');
-Before(({ I }) => { // or Background
-    I.loginAsVendor();
-});
-Scenario('Group Product Functional Behaviour Testing', ({ I }) => {
+// Before(({ I }) => { // or Background
+//     I.loginAsVendor();
+// });
+Scenario('Group Product Functional Behaviour Testing', ({ I,loginAs }) => {
+    loginAs('vendor');
```

Comment:
Maintain upper and lower case for the user role because the user is case sensitive.